tests/transforms/test_qcut.py

```diff
@@ -1799,3 +1803,39 @@ def f(x):
         ) * f(1)
 
         assert np.allclose(grad, expected_grad)
+
+
+class TestCutCircuitTransform:
+    """
+    Tests for the cut_circuit transform
+    """
+
+    def test_simple_cut_circuit(self, mocker):
+        """
+        Tests the full circuit cutting pipeline returns the correct value and
+        gradient for a simple circuit using the `cut_circuit` transform.
+        """
+
+        dev = qml.device("default.qubit", wires=2)
+
+        @qml.qnode(dev)
+        def circuit(x):
+            qml.RX(x, wires=0)
+            qml.RY(0.543, wires=1)
+            qml.WireCut(wires=0)
+            qml.CNOT(wires=[0, 1])
+            qml.RZ(0.240, wires=0)
+            qml.RZ(0.133, wires=1)
+            return qml.expval(qml.PauliZ(wires=[0]))
+
+        spy = mocker.spy(qcut, "qcut_processing_fn")
+        x = np.array(0.531, requires_grad=True)
+        cut_circuit = qcut.cut_circuit(circuit)
+
+        assert round(cut_circuit(x), 8) == round(float(circuit(x)), 8)
```

Comment:
maybe we should be returning a tensor with `requires_grad` for the cut circuit?

Reply:
Yes need to look into how things are cast. I wouldn't worry for this PR, as long as the grad and functions are identical